modules/custom/az_news/az_news_marketing_cloud/az_news_marketing_cloud.routing.yml

```diff
@@ -0,0 +1,8 @@
+az_news_marketing_cloud.template:
+  path: '/export/marketing-cloud'
+  defaults:
+    _controller: '\Drupal\az_news_marketing_cloud\Controller\PageOutputController'
```

Comment:
Is this controller still used in this PR?

Reply:
This file still exists.. think we can/should delete it?
https://github.com/az-digital/az_quickstart/pull/2360/files#diff-d099244a8af8e24e4e8e2510b3daf1e0e5d2a16d07a087651fb73f752386c95e

Reply:
I think so. It seems like these might be from an earlier version of this PR. `modules/custom/az_news/az_news_marketing_cloud/src/Controller/PageOutputController.php` and `modules/custom/az_news/az_news_marketing_cloud/az_news_marketing_cloud.routing.yml` don't seem used anymore.

`PageOutputController` seems to whitescreen if it's actually called and doesn't output anything as far as I can tell. It looks like all of the output comes from the `az_marketing_cloud` view currently.

Reply:
Looks safe to delete when I tested locally